templates/model_utils.mustache

```diff
@@ -0,0 +1,1602 @@
+{{>partial_header}}
```

Comment:
What is this file? Is it for the 2 secu issues ?

Reply:
Yes it was missing. We didn't have to add it before because we didn't have to change the template.

Reply:
and what have we changed in the template?